src/components/Menu.vue

```diff
@@ -1,5 +1,18 @@
 <template>
   <div id="controls">
+    <span
+      class="nomlogSummary"
+      v-show="session.voteHistory.length"
+      @click="toggleModal('voteHistory')"
+      :title="
+        `${session.voteHistory.length} recent ${
+          session.voteHistory.length == 1 ? 'nomination' : 'nominations'
```

Comment:
```suggestion
        `${session.voteHistory.length} recent 
          nomination${session.voteHistory.length > 1 ? 's' : ''
```

Reply:
I'd prefer to not change this: In English, singular case is only used for 1 item, plural is used for 0 as well. That doesn't matter *right now*, but it does make changing policy about this item (and translation of the app) easier.

Reply:
Well, given that the icon is only visible for voteHistory.length >= 1, using the plural for 0 would be moot, but I see your point. 👍